style.css

```diff
@@ -0,0 +1,66 @@
+header {
+    display: block;
+    position: fixed;
+    left: 0px;
+    width: 100%;
+    height:20px;
```

Comment:
make sure to be consistent, in that it's always best to put a space after the colon in CSS!